src/app/pages/page-not-found/page-not-found.component.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  text-align: center;
+}
```

Comment:
```suggestion
```

We don't center card titles on other error cards/dialogs. (Though, if you really want to you should use our `centered-text` class.)